Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/server/DataDonationAnalyticsServer.kt

```diff
@@ -0,0 +1,30 @@
+package de.rki.coronawarnapp.datadonation.analytics.server
+
+import dagger.Lazy
+import de.rki.coronawarnapp.datadonation.analytics.AnalyticsException
+import de.rki.coronawarnapp.server.protocols.internal.ppdd.PpaDataRequestAndroid
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class DataDonationAnalyticsServer @Inject constructor(
+    private val api: Lazy<DataDonationAnalyticsApiV1>
+) {
+
+    suspend fun uploadAnalyticsData(ppaDataRequestAndroid: PpaDataRequestAndroid.PPADataRequestAndroid) {
```

Comment:
consider to use 'when' instead of various if statements if or when you want (optional)

Reply:
done :)